docs/source/commandline/commandline-all.rst

```diff
@@ -1715,45 +2406,53 @@ Scripting Options
 
 **-autoboot_command** *"<command>"*
 
-	Command string to execute after machine boot (in quotes " "). To issue a quote to the emulation, use """ in the string. Using **\\n** will issue a create a new line, issuing what was typed prior as a command.
+    Command string to execute after machine boot (in quotes " "). To issue a
+    quote to the emulation, use """ in the string. Using **\\n** will issue a
```

Comment:
This has shell escaping mixed up with what MAME interprets.  Can be fixed later though.